virt/virt-4-10-release-notes.adoc

```diff
@@ -117,6 +117,8 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 * {VirtProductName} has xref:../virt/logging_events_monitoring/virt-virtualization-alerts.adoc#virt-virtualization-alerts[critical alerts] that inform you when a problem occurs that requires immediate attention. Now, each alert has a corresponding description of the problem, a reason for why the alert is occurring, a troubleshooting process to diagnose the source of the problem, and steps for resolving the alert.
 
+* A cluster administrator can now back up namespaces containing VMs by using the xref:../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[OpenShift API for Data Protection] with the `kubevirt` plugin.
```

Comment:
s/containing/that contain

s/plugin/plug-in

(IBM Style Guide)
plug-in   noun, adjective
Write as shown: hyphenated.

plugin   noun, adjective
Do not use. Use "plug-in".